README.md

```diff
@@ -299,17 +299,18 @@ on the [Flutter contributors Discord][]!
 
 A build that fails with the error
 `Error: Some code excerpts needed to be updated!`
-means that one or more code excerpts in the markdown file
-aren't identical to the
-code in the corresponding `.dart` file.
+means that one or more code excerpts in the site Markdown files
```

Comment:
**suggestion:** Maybe this should clarify that the `*.dart` file is the definitive version of the code and running this tool overwrites what's in the `*.md` file with the code from the linked excerpt?